### DISCUSSION TITLE
Network: use --host to expose

### DISCUSSION THREAD

**Discussion by mwaqashameed** [kriasoft/react-starter-kit#2024] [Q&A]
When I try to run with "yarn start" it is giving following error.
  ➜  Local:   http://localhost:5173/
  ➜  Network: use --host to expose

can you help me out ? 
Thanks,

**Comment by koistya** [comment]
It's not an error. That's how [Vite](https://vitejs.dev/) launches the app and shows the URL. You're expected to open that URL in the browser.

Note, `yarn start` is a shortcut for the `yarn workspace app dev` Vite command. You can check the full list of Vite CLI options like this:

```
$ yarn workspace app vite --help
```

For example, you can launch the app and make it open the browser by adding `--open` CLI argument:

```
$ yarn workspace app vite dev --open
```

https://vitejs.dev/guide/cli.html
